src/modules/upload/index.js

```
@@ -338,14 +336,48 @@ const readAllEntries = async dirReader => {
+    ? getFullpath(client, dirID, name, driveId)
+    : CozyFile.getFullpath(dirID, name)
+
+const getExistingDirectory = async (client, dirID, name, driveId) => {
+  const path = await resolveFullpath(client, dirID, name, driveId)
+  const statResp = await client
+    .collection(DOCTYPE_FILES, { driveId })
+    .statByPath(path)
+  if (statResp.data.type !== 'directory') {
+    const err = new Error(
+      `"${path}" already exists and is not a directory`
+    )
```

ESLint

[error] Replace `⏎······`"${path}"·already·exists·and·is·not·a·directory`⏎····` with ``"${path}"·already·exists·and·is·not·a·directory``

(prettier/prettier)